draft-ietf-httpbis-digest-headers.md

```diff
@@ -334,11 +321,11 @@ Response:
 
   SHA:
   :  
-    * Description:  The SHA-1 algorithm [FIPS180-1].  The output of this
```

Comment:
Strictly, SHA-1 is specified in https://tools.ietf.org/html/rfc3174 

I got there by following the link from RFC 6234, both are still active RFCs.

Reply:
3174 is updated by 6234. Do you think we should change the PR and reference both?

Reply:
Personally I'd like to reference both.

"Updates" or "Updated by" is additive (in contrast, "Obsoletes" is replacement) so both documents are still active and IMO if we reference 3174 we avoid one level of indirection for people that want to see the active spec for SHA-1.

Reply:
should be fixed now, WDYT?